exponent_server_sdk/__init__.py

```diff
@@ -292,7 +292,8 @@ def _publish_internal(self, push_messages):
                 # If there is no status, assume error.
                 status=receipt.get('status', PushResponse.ERROR_STATUS),
                 message=receipt.get('message', ''),
-                details=receipt.get('details', None)))
```

Comment:
Replacing 'details' for 'id' also works. I don't think details is returning anything.